recipes/nnlojet/recipe.yaml

```diff
@@ -0,0 +1,93 @@
+context:
+  name: "nnlojet"
+  version: "1.0.0"
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://nnlojet.hepforge.org/nnlojet-v${{ version }}.tar.gz
+  sha256: b02013922387cf37745487d23b271eaae257c8af76bb3cca6bbf885dd8139e74
+
+build:
+  number: 0
+  script:
+    # Get an updated config.sub and config.guess
+    - cmake "${CMAKE_ARGS}" -DCMAKE_INSTALL_PREFIX="${PREFIX}" -S . -B build
```

Comment:
The error that is happening here seems similar to https://gitlab.kitware.com/cmake/cmake/-/issues/21454. I haven't sat down to poke at this directly, but I think I'd need @henryiii to comment on what's actually happening with https://github.com/conda-forge/staged-recipes/pull/30272#issuecomment-2956357982.